src/ostorlab/apis/runners/authenticated_runner.py

```diff
@@ -57,17 +56,18 @@ def __init__(self,
         self._username = username
         self._password = password
         self._token_duration = token_duration
-        self._api_key: Optional[str] = self._configuration_manager.get_api_key()
+        if api_key:
```

Comment:
this can be simplified to:
self._api_key = api_key or self._configuration_manager.get_api_key()